source/blog/2019-05-21-monetizing-open-source.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Monetizing Open Source"
+authorName: Jonathan H. Wage
+authorEmail: jonwage@gmail.com
+permalink: /2019/05/21/monetizing-open-source.html
+---
+
+In our quest to make Doctrine financially sustainable, we have created
+Doctrine Company, LLC for the team to do business under. We've had a strong
+10 years of open source success but we believe that in order for Doctrine to be
+truly sustainable and to last another decade, we need to diversify and monetize
+the project. It is our goal to be able to generate enough money through the project
+to fund full or part time work on the project. Below you will find details of some
+of our monetization efforts.
+
+## Patreon
+
+Patreon is a platform that allows open-source maintainers, artists, creators, etc. to create
+relationships with their users and offer them benefits for becoming a Patreon.
+
+If you would like to make a donation to the project, Patreon is the place to do it.
+Depending on the tier, we offer a few different benefits:
+
+- A mention on Twitter.
+- Your name and website link on the Doctrine website.
+- Priority responses to Stackoverflow or GitHub Issues/Pull Requests.
+- Placement on a Partners page on the Doctrine website, with your logo, link and a description advertising your company's services.
+
+[Become a Patreon](https://www.patreon.com/phpdoctrine) of Doctrine today!
+
+## Consulting
+
+Under the Doctrine Company, we are providing [Consulting and Training](/consulting.html)
+services to companies that use PHP. The services we offer are not limited to Doctrine itself.
+Our team specializes in writing high quality PHP. Whether you are starting a new project or
```

Comment:
"producing" might be a more enticing word than "writing". I'm pretty sure @Ocramius doesn't use a keyboard at this point and metaphysically conjures source code into existence with his mental powers.